playbooks/roles/ocp-cli-plugins/tasks/custom-cli-plugin.yml

```
@@ -0,0 +1,71 @@
+---
+
+## Openshift custom-cli plugin test
+
+- name: Custom-cli plugin test
+  block:
+    - block:
+        - name: Install the user defined custom-plugin
+          copy:
+            src: "{{ role_path }}/files/{{ user_custom_plugin }}"
```

Can we use actual shell script file here, instead of text file? may be we can reduce redundant tasks.

```suggestion
            src: "{{ role_path }}/files/{{ user_custom_plugin | default('oc-foo.sh', true) }}"
```